### ISSUE TITLE
Add [aria-label] to the <nav> in menu.html

### ISSUE THREAD

**Issue by tajakobsen** [enonic/lib-menu#39]
The `<nav>` element in *menu.html* should have an `[aria-label]` that can be configured from `getMenuTree()` and `getSubMenus()`.

My suggestion to achieve this is the following:

1. Add a `navigationAriaLabel` field to the `params` config-objects of both functions
2. Make the functions return an `Object` instead of `Arrays` of menu-items.
   - The object should have this shape `{ navigationAriaLabel?: string, menuItems: Array<MenuItem>  }`
   - This also opens up the opportunity to send more data in this object from the js-code to Thymeleaf in the future
3. Change the Thymeleaf-template to take the object from 2 as input, instead of an array.
4. Add the `data-th-aria-label="${menu.navigationAriaLabel}"` to the `<nav>` in *menu.html*

### Breaking changes:

Changing the signatures of `getMenuTree()` and `getSubMenus()`, is a breaking change, since they now will return objects instead of arrays. So the change described here should only be included in a *major version bump*.

**Comment by poi33** [comment]
Attempted to add this in bd59bda5560c456123f2b1289de8e767c6cc70e1.
